const.py

```diff
@@ -14,4 +16,5 @@ class IDENTITY_FIELDS:
 class CONFIG_FIELDS:
     SEARCH = "search"
     COUNT = "count"
-    COUNT_INCREMENT = 10
```

Comment:
CONFIG_FIELDS should not be used for other stuff, just to be consistent with conventions. 
CONFIG_FIELDS is specifically used for USER_INPUT_JSON fields